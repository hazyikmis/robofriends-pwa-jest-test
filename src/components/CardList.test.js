import { shallow } from "enzyme";
import React from "react";
import CardList from "./CardList";

it("snapshot match testing: CardList component", () => {
  const mockRobots = [
    {id: 1, name: "John Snow", username: "john", email: "john@snow.com"},
    {id: 2, name: "Regina Teideman", username: "jregina", email: "regina@teideman.com"}
  ]
  expect(shallow(<CardList robots={mockRobots} />).debug()).toMatchSnapshot();
})