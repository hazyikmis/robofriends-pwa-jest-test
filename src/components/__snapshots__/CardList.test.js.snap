// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot match testing: CardList component 1`] = `
"<div>
  <Card id={1} name=\\"John Snow\\" email=\\"john@snow.com\\" />
  <Card id={2} name=\\"Regina Teideman\\" email=\\"regina@teideman.com\\" />
</div>"
`;
